src/components/filter/LessonFilter.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import FilterGroup from './FilterGroup';
+import Divider from 'material-ui/Divider';
+import {Card, CardText, CardTitle} from 'material-ui/Card';
+
+const LessonFilter = React.createClass({
+  render(){
+    const tagGroups = this.props.tagGroups;
+    const tagGroupNodes = tagGroups.map((tagGroup, idx) => {
```

Comment:
Hvis du bestemmer deg for å la tagGroups beholde strukturen til lessonTagGroups, så kan du jo bruke
```
const tagGroupNodes = Object.keys(tagGroups).map((tagGroup, idx) => {
  ...
});